### PR TITLE
For #9062, reinstate status.

### PR DESCRIPTION
**Pull request by gplsteph**
Revisited a couple of things.
Added setting reinstated Shot status.
Added a param to not update Shots. (not sure if this should be driven by a settings or just a command line param?)

### CODE REVIEW

File: sg_otio/cut_diff.py [hunk 1]
```diff
@@ -338,7 +338,7 @@ def _check_and_set_changes(self):
         # We have both a Shot and a current clip.
         if self.sg_shot_status and self.sg_shot_status.get("code"):
             # TODO: check if matching by short code is right? #9320
```
Comment (ksallee) [inline sg_otio/cut_diff.py:340]: remove this, no?

File: sg_otio/sg_settings.py [hunk 2]
```diff
@@ -347,22 +348,58 @@ def shot_cut_fields_prefix(self, value):
...
+        Return the SG Shot status to use when reinstating Shots.
+
+        :returns: A SG status short code or ``None``.
+        """
+        return self._shot_reinstate_status
+
+    @shot_reinstate_status.setter
+    def shot_reinstate_status(self, value):
+        """
+        Set the SG Shot status to use when omitting Shots.
```
Comment (ksallee) [inline sg_otio/sg_settings.py:380]: maybe add something about the magic constant "Previous Status" and how it works?

**Review by ksallee — DISMISSED** [review]
LGTM

File: sg_otio/cut_diff.py (hunk 1 above)
Comment (gplsteph) [reply sg_otio/cut_diff.py:340]: Let's keep it and remove when we do 9320 if you don't mind.

**Pull request merged**